### PR TITLE
PMM-7380 backup location list add

### PR DESCRIPTION
**Pull request by Dasio**
[PMM-7380](https://jira.percona.com/browse/PMM-7380)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1458

- [ ] Links to other linked pull requests (optional).
- API: https://github.com/percona/pmm/pull/674
- PMM: https://github.com/percona/pmm-managed/pull/644

### CODE REVIEW

File: management/backup/locations_test.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+		assert.NotEmpty(t, resp.Payload.LocationID)
+	})
+}
+
+func TestAddWrongLocation(t *testing.T) {
+	client := backupClient.Default.Locations
+	t.Parallel()
+
+	t.Run("missing config", func(t *testing.T) {
```
Comment (BupycHuk) [inline management/backup/locations_test.go:65]: case with both configs passed is missed

**Review by BupycHuk — COMMENTED** [review]
Please require review from one more developer

File: management/backup/locations_test.go [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func TestAddLocation(t *testing.T) {
+	client := backupClient.Default.Locations
+	t.Parallel()
```
Comment (artemgavrilov) [inline management/backup/locations_test.go:18]: Let's keep `t.Parralel` on the first line of the test function

**Review by artemgavrilov — APPROVED** [review]

File: management/backup/locations_test.go [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+			assert.Equal(t, body.FsConfig.Path, loc.FsConfig.Path)
+			found = true
+		}
+	}
+	assert.True(t, found, "Expected location not found")
+}
+
+func deleteLocation(t *testing.T, client locations.ClientService, id string) {
+	t.Helper()
+	// @TODO
```
Comment (BupycHuk) [inline management/backup/locations_test.go:189]: please add link to the task

**Review by BupycHuk — APPROVED** [review]

File: Gopkg.toml [hunk 4]
```diff
@@ -10,7 +10,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
```
Comment (oter) [inline Gopkg.toml:13]: `release/2.15` should be here, that's why you are having changes in vendor folder like start_pt_mongo_db_summary_action_parameters.go etc.

Comment (Dasio) [reply Gopkg.toml:13]: but i need my branch with my changes

Comment (oter) [reply Gopkg.toml:13]: you changed the base branch from `master` to `release/2.15`, but didn't pull new changes from `release/2.15`

**Pull request merged**